pkg/component/registrycaches/registry_caches.go

```diff
@@ -18,6 +18,7 @@ import (
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
 	v1beta1helper "github.com/gardener/gardener/pkg/apis/core/v1beta1/helper"
+	"github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
```

Comment:
Nit:

```suggestion
	resourcesv1alpha1 "github.com/gardener/gardener/pkg/apis/resources/v1alpha1"
```

PS: We should enable the import alias linter as done in gardener/gardener. (cc @dimitar-kostadinov)